README.md

```diff
@@ -1,11 +1,14 @@
 # ExpressJS Template
 
+| INDEX |
+|:-:|
+| [Setup Project](#Setup-Project) |
+| [Project Architecture Overview](#Project-Architecture-Overview) |
```

Comment:
The index should be a numbered list with hrefs that point to the titles/subtitles.

Reply:
@rendoir Would you mind making those changes for me please?